src/yang/ietf-bgp-rib-types.yang

```diff
@@ -163,6 +162,6 @@ submodule ietf-bgp-rib-types {
   identity rejected-import-policy {
     base bgp-not-selected-policy;
     description
-      "Route was rejected after apply import policies";
+      "Route was rejected after applying import policies";
```

Comment:
Missing a period at the end.

Reply:
Added period, but see other review comment replies about our inconsistent punctuation.

Reply:
Agreed. But since we are updating the comments, we can address punctuation. Did see your issue on punctuation which we can address for remaining descriptions.